LUFA/Drivers/USB/Class/Common/HIDClassCommon.h

```diff
@@ -330,6 +330,23 @@
 		#define HID_KEYBOARD_SC_RIGHT_SHIFT                       0xE5
 		#define HID_KEYBOARD_SC_RIGHT_ALT                         0xE6
 		#define HID_KEYBOARD_SC_RIGHT_GUI                         0xE7
+		#define HID_KEYBOARD_SC_MEDIA_PLAY                        0xE8
```

Comment:
How about `HID_CONSUMER_MEDIA_*` or something similar instead? The consumer page contains more than just usages for media control, and a device which uses that page is not necessarily a keyboard.